### ISSUE TITLE
question

### ISSUE THREAD

**Issue by ddxoxpp** [riscv/riscv-isa-manual#1641]
In the out of order cpu implementation, when an exception occurs in any element, can I set vstart directly to 0?

**Comment by aswaterman** [comment]
No.  `vstart` must be set to the index of the element that faulted.
